metaconfig-core/shared/src/main/scala/metaconfig/internal/CanBuildFromDecoder.scala

```diff
@@ -57,6 +65,21 @@ object CanBuildFromDecoder {
           )
           NotOk(error)
       }
-    }
 
+      override def readWithDefault(
+          conf: Conf,
+          default: C[A]
+      ): Configured[C[A]] = {
+        conf match {
+          case Conf.Obj(("add", conf) :: Nil) =>
```

Comment:
can we use a symbol for that instead? something like
```
align.tokens.+ = [ ... ]
align.tokens.- = [ ... ] // this wouldn't hurt either
```

Reply:
HOCON has `+=` [operator](https://github.com/lightbend/config/blob/master/HOCON.md#the--field-separator) but its full support will be very difficult. So I don't want to use `+` in order not to create confusion with not-supported HOCON ops.

Reply:
It's still possible to use `+=` with Metaconfig but it would require a change in how we distribute the base Scalafmt configuration. The `+=` HOCON operator works when users can import the base configuration somehow like `include scalafmt.conf`. There's nothing on the Metaconfig side that we need to change to support `+=`.

Reply:
I mean that `+=` in HOCON will not append to default values described in Scala classes

Reply:
Correct. It’s possible to achieve that if we encode the Scalafmt config first as HOCON and then import those settings from the HOCON